prow/github/report/report.go

```diff
@@ -306,3 +301,47 @@ func createComment(reportTemplate *template.Template, pj prowapi.ProwJob, entrie
 	}...)
 	return strings.Join(lines, "\n"), nil
 }
+
+// getOldEntries processes a comment body and obtains a list of entries.
+func getOldEntries(body string) []string {
```

Comment:
Why not store machine-readable content hidden from users?

Reply:
i'm not a fan of the flatten, unflatten but i wanted to keep this PR scoped.
it kind of already went out of scope by extracting these get*Entries functions out. :)

Reply:
/shrug